docs/commands.md

```diff
@@ -106,6 +107,9 @@ Global Flags:
 
 The other commands have no further options.
 
+## Exclude Paths
+By default, KICS now automatically excludes paths according to the .gitignore file. If you want to disable this behavior, you use the flag `--add-gitignore`.
```

Comment:
Change to
By default, KICS excludes paths specified in the .gitignore file in the root of the repository. To disable this behavior, use flag `--add-gitignore`.